apps/apps_service.go

```diff
@@ -47,6 +47,8 @@ func (svc *appsService) CreateApp(name string, pubkey string, maxAmountSat uint6
 
 	if budgetRenewal == "" {
 		budgetRenewal = constants.BUDGET_RENEWAL_NEVER
+	} else {
+		budgetRenewal = strings.ToLower(budgetRenewal)
```

Comment:
This doesn't follow the spec, I would rather throw an error which we do below

Reply:
Hmmm yeah I think it should be debuggable from the error, removing this :+1: